lib/theme_check/regex_helpers.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ThemeCheck
+  module RegexHelpers
+    def matches(s, re)
```

Comment:
IIUC what this code does, you're looking for `String#scan`: https://ruby-doc.org/core-3.0.0/String.html#method-i-scan

Reply:
You can use `https://ruby-doc.org/core-3.0.0/Regexp.html#method-c-last_match`